app/components/Chatbot/index.tsx

```diff
@@ -14,15 +14,15 @@ import {useSearch} from '~/hooks/useSearch'
 const poolQuestions = [
   {
     title: 'What is AI Safety? - from pool',
-    pageid: '1b',
+    pageid: '5633',
   },
   {
     title: 'How would the AI even get out in the world? -- from pool',
-    pageid: '2b',
+    pageid: '7638',
   },
   {
     title: 'Do people seriously worry about existential risk from AI? --- from pool',
-    pageid: '3b',
+    pageid: '7639',
```

Comment:
sorry to be a pain - could you add like 10 extra questions here and use a random selection of them?